README.md

```diff
@@ -35,14 +33,53 @@ Environments under Python 2.7.12 and 3.6.0 are tested.
 
 # Features
 
+## Models
+Currently, ChainerCV supports networks for object detection and image segmentation for 2D images.
+Image detection is the task of finding where objects are in an image, and classify the class to which the object belongs to.
+Semantic segmentation is the task of segmentingimages into pieces and assigning object labels to them.
+Our implementations include:
+
++ **Faster RCNN**
++ **Single Shot Multibox Detector (SSD)**
++ **SegNet**
+
+The models support common interface across architectures among each task.
+For example, detection models support method that takes images and outputs coordinates, class labels and confidence scores of bounding boxes predicted around estimated regions of objects.
+The common interface allows users to swap different models easily inside their code.
+On top of that, the utilitiy codes build on top of this interface.
+For example, there is an extension `chainercv.extensions.detection_vis_report` that visualizes outputs during training for models that previously stated common interface.
```

Comment:
`extension` -> `extension of chainer.training.Trainer`
`chainercv.extensions.detection_vis_report` -> `chainercv.extensions.DetectionVisReport`